recipe/meta.yaml

```
@@ -42,7 +42,6 @@ requirements:
     - python
     - protobuf
```

Add the correct protobuf pins and remove the patch in line 10 (incl. the patch file)
```suggestion
    - protobuf >=3.20.2,<4
```